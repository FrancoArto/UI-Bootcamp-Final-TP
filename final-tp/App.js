import React from 'react';
import { Provider } from 'react-redux';
import createStore from './app/store/store'
import './app/reactotronConfig'
import { AppNavigator } from './app/components/AppNavigator';


const store = createStore

class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <AppNavigator />
      </Provider>
    );
  }
}

export default App;