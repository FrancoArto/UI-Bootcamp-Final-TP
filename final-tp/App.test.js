import React from 'react';
import App from './App';
import ShallowRenderer from 'react-test-renderer/shallow';


it('renders without crashing', () => {
  const rendered = new ShallowRenderer()
  rendered.render(<App />);
  expect(rendered).toMatchSnapshot();
});

