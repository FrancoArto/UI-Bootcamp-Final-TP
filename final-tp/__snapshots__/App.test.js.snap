// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_element": <App />,
  "_forcedUpdate": false,
  "_instance": App {
    "context": Object {},
    "props": Object {},
    "refs": Object {},
    "state": null,
    "updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
  },
  "_newState": null,
  "_rendered": <Provider
    store={
      Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <AppNavigator />
  </Provider>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
}
`;
